web/src/confirm_dialog.ts

```diff
@@ -4,10 +4,9 @@ import {$t_html} from "./i18n";
 
 export function launch(conf: DialogWidgetConfig): void {
     dialog_widget.launch({
-        ...conf,
         close_on_submit: true,
         focus_submit_on_open: true,
         html_submit_button: $t_html({defaultMessage: "Confirm"}),
-        // Used to control button colors in the template.
```

Comment:
I'm unsure if this comment was referring to something -- should we keep it?

Reply:
It's easy to answer these questions:
```
git log -S"Used to control button colors in the template"
```

Finds 6fb0d414b43b5a9f6a9bef22a0298ae5e6f29dcc, which doesn't give any hint as to what parameters were supposed to be below this. I think it must have been intended for `danger_submit_button`, which was removed.